HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -72,35 +79,82 @@ public MapObjectEntity DisplayMapObject
             }
         }
 
-        public MainWindow()
+        private MapObject _selectedMapObject;
+
+        public MapObject SelectedMapObject
```

Comment:
What is the purpose of DisplayMapObject, and what is the purpose of SelectedMapObject? I can't tell from their names.

Reply:
i didn't write that

Reply:
How did you add these lines of code then?